src/components/switch/bl-switch.css

```diff
@@ -0,0 +1,72 @@
+:host {
+  display: inline-block;
+  cursor: pointer;
+}
+
+span {
+  display: inline-block;
+}
+
+.switch {
+  --track-height: var(--bl-size-xl, 24px);
+  --track-width: var(--bl-size-4xl, 48px);
+  --thumb-offset: var(--bl-size-4xs, 2px);
+  --thumb-height: calc(var(--track-height) - calc(2 * var(--thumb-offset)));
+  --thumb-width: var(--thumb-height);
+  --outline-width: var(--bl-size-4xs, 2px);
```

Comment:
`--bl` prefixed variables are our tokens and we assume that they always set on the document. So you don't need to set default values for them. This will prevent to duplicate same values on many places.